services/kiali/2.7.1/extra-images.txt

```diff
@@ -0,0 +1 @@
+quay.io/kiali/kiali:v2.7.1-distro
```

Comment:
Could you add dynamic replacement for the version? Similar to this - https://github.com/mesosphere/kommander-applications/blob/main/services/harbor/1.16.2/database/extra-images.txt